python/utils.py

```diff
@@ -526,7 +532,7 @@ def showPluginHelp(packageName=None, filename="index", section=""):
         url = "file://" + helpfile
         if section != "":
             url = url + "#" + section
-        iface.openURL(url, False)
+        QDesktopServices.openUrl .openUrl(url, False)
```

Comment:
Looks like this line got mangled during the commit

Reply:
Ups I misspelled the more important line!. Fixed.

Thanks to check it.